Source/MIPSAssembler.cpp

```diff
@@ -42,15 +42,14 @@ void CMIPSAssembler::CreateLabelReference(LABEL label)
 
 void CMIPSAssembler::ResolveLabelReferences()
 {
-	for(LabelReferenceMapType::iterator labelRef(m_labelReferences.begin());
-		m_labelReferences.end() != labelRef; labelRef++)
+	for (auto& labelRef : m_labelReferences)

```

Comment:
Please keep no spaces between `for` and parenthesis. Also, I think `labelRef` could be made `const`.